### PR TITLE
Update otto extractor

### PR DESCRIPTION
**Pull request by BigDatalex**
Otto is changing their schema.org tagging from microdata and dublincore to JSON-LD. This PR makes the required changes while allowing for backwards compatibility.

### CODE REVIEW

File: extract/extract/extractors/otto_de.py [hunk 1]
```diff
@@ -34,26 +35,34 @@ def extract_otto_de(parsed_page: ParsedPage) -> Optional[Product]:
     microdata = safely_return_first_element(parsed_page.schema_org.get(MICRODATA, [{}]))
     properties = microdata.get("properties", {})
 
-    name = properties.get("name", None)
-    brand = properties.get("brand", None)
+    name = properties.get("name")
```
Comment (felixbiessmann) [inline extract/extract/extractors/otto_de.py:41]: What's the default return value when not specifying it here? Would the get request fail?

Comment (BigDatalex) [reply extract/extract/extractors/otto_de.py:41]: The default is `None`, so it is not necessary to specify it. If the request fails, it just returns `None`.

**Comment by BigDatalex** [comment]
@AdriaSG have you seen, that I requested your review on this?

**Review by AdriaSG — APPROVED** [review]
Looks good to me :)

**Pull request merged**